sentinel-adapter/sentinel-web-servlet/src/main/java/com/alibaba/csp/sentinel/adapter/servlet/config/WebServletConfig.java

```diff
@@ -41,4 +47,31 @@ public static String getBlockPage() {
     public static void setBlockPage(String blockPage) {
         SentinelConfig.setConfig(BLOCK_PAGE, blockPage);
     }
+
+    /**
+     * Return status 429 in the default block page,
+     * you can use -Dcsp.sentinel.web.servlet.block.page.http.status=200 or other http status,
+     * to set http status which you want of the default block page.
+     * When csp.sentinel.web.servlet.block.page.http.status is empty or not number,
+     * the block page http status will be automatically set to 429(Too Many Requests).
+     *
+     * @return block page http status
+     */
+    public static int getBlockPageHttpStatus() {
+        String value = SentinelConfig.getConfig(BLOCK_PAGE_HTTP_STATUS);
+        if (StringUtil.isEmpty(value)) {
+            setBlockPageHttpStatus(HTTP_STATUS_TOO_MANY_REQUESTS);
+            return HTTP_STATUS_TOO_MANY_REQUESTS;
+        }
+        try {
+            return Integer.parseInt(SentinelConfig.getConfig(BLOCK_PAGE_HTTP_STATUS));
+        } catch (NumberFormatException e) {
+            setBlockPageHttpStatus(HTTP_STATUS_TOO_MANY_REQUESTS);
```

Comment:
Maybe we could also add a log here, like `RecordLog.warn("[WebServletConfig] Invalid block HTTP status {0}, using default 429", s)`

Reply:
I'll improve this by myself later.

Reply:
> I'll improve this by myself later.

I saw this message, sorry

Reply:
I've polished the code in https://github.com/alibaba/Sentinel/commit/9d514d50365e4a9c8b4d6cb7d525e9cbc49c5364.

Besides, the previous block status property name might be too long, so I simplified it to `csp.sentinel.web.servlet.block.status`.